src/script/lua_api/l_util.cpp

```diff
@@ -219,18 +219,14 @@ int ModApiUtil::l_write_json(lua_State *L)
 	return 1;
 }
 
-// get_dig_params(groups, tool_capabilities[, time_from_last_punch])
+// get_dig_params(groups, tool_capabilities)
 int ModApiUtil::l_get_dig_params(lua_State *L)
```

Comment:
Mind documenting `get_hit_params` and `get_dig_params` in the Lua API or does that rather belong into a new pull/issue?